### PR TITLE
feature: Added ability to output graphs as svg

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: html/includes/graphs/graph.inc.php [hunk 1]
```diff
@@ -102,6 +102,10 @@ function graph_error($string)
     }
 } else {
     // $rrd_options .= " HRULE:0#999999";
+    if ($config['webui']['graph_type'] === 'svg') {
+        $rrd_options .= " --imgformat=SVG -m 0.75 -W 'librenms'";
```
Comment (murrant) [inline html/includes/graphs/graph.inc.php]: Why add the librenms text at the bottom?

Comment (laf) [inline html/includes/graphs/graph.inc.php]: That was just debug

**Comment by laf** [comment]
I'll try and get those docs together tomorrow.

**Comment by laf** [comment]
All done

**Comment by FTBZ** [comment]
Thanks, look very nice on my 4K/5K displays.

**Pull request merged**